### DISCUSSION TITLE
Google Colab notebook to run QCxMS in the cloud with Python notebook

### DISCUSSION THREAD

**Discussion by tobigithub** [qcxms/QCxMS#45] [General]
Here is an interesting showcase to bring QCxMS to the masses, no installation or pre-knowledge required, only a Google Colab login, which is free and comes with a Xeon 2.2 Ghz (1 CPU core) and 12h runtime I think. 

**Link:** [Colab for in silico mass spectrometry via quantum and moleculary dynamic calculations](https://colab.research.google.com/drive/18LqKvIW9iPi5BFI6yyEYtzn1czMOHeX2?usp=sharing)
**Author:**  @dimenwarper

The current version (Oct 2021) currently does not calculate protonated ions for CID, so one can start with an charged ion like protonated urea, which is also small enough.  Protonated urea: "[H]N([H])C(=[OH+])N([H])[H]"

For larger molecules the Python Notebook enabled [Google Datalab](https://cloud.google.com/datalab/docs) is also available, but that's not free. Overall great concept, because even generating an XYZ file or analyzing trajectories could be a large hurdle for new users. This QCxMS Colab notebook makes everything so much easier, of course can be extended to many more options, but this is a great starter.

**Comment by tobigithub** [comment]
New GitHub for the above project:
https://github.com/dimenwarper/colab_biowrappers/blob/main/ColabQCxMS.ipynb
Author: @dimenwarper

**Reply by JayTheDog** [reply]
Indeed very interesting! Thanks for sharing!
